src/bin/pg_autoctl/cli_drop_node.c

```diff
@@ -395,7 +400,12 @@ cli_drop_node(int argc, char **argv)
 			exit(EXIT_CODE_BAD_ARGS);
 		}
 
-		(void) cli_drop_node_from_monitor(&config);
+		int64_t nodeId = -1;
+		int groupId = -1;
+
+		(void) cli_drop_node_from_monitor(&config, &nodeId, &groupId);
+
+		(void) wait_until_node_is_dropped(&config, nodeId, groupId);
```

Comment:
I think it would be good to have a log here, or at the end of `wait_until_node_is_dropped` saying something like: "Node dropped successfully"

Reply:
Good point, added now.